include/Webservices/Utils.php

```diff
@@ -442,6 +444,7 @@ function vtws_addWebserviceOperationParam($operationId, $paramName, $paramType,
  */
 function vtws_getModuleHandlerFromName($name, $user)
 {
+	global $log;
```

Comment:
Global variable or function should never be used
https://insight.sensiolabs.com/projects/189f9068-d777-44a7-9164-9242e81df88c/analyses/163#rule-24-038